gensim/models/word2vec.py

```diff
@@ -329,7 +333,8 @@ def train_sg_pair(model, word, context_index, alpha, learn_vectors=True, learn_h
     return neu1e
 
 
-def train_cbow_pair(model, word, input_word_indices, l1, alpha, learn_vectors=True, learn_hidden=True, compute_loss=False,
+def train_cbow_pair(model, word, input_word_indices, l1, alpha, learn_vectors=True, learn_hidden=True,
```

Comment:
Please use hanging indents (instead of vertical), here and anywhere

Reply:
"When using a hanging indent the following should be considered; there should be no arguments on the first line and further indentation should be used to clearly distinguish itself as a continuation line."

Is this what you need ? if yes, do you recommend any tool other than autopep8 that would auto format the file ?

Reply:
Sorry, incorrect line
vertical indents OK for function/method definition, that's all. For other situations, in gensim, we used hanging indent.

Unfortunately, I can't recommend any tool for it (because we have no tool for check this condition yet), only manually